src/auth/auth.spec.ts

```diff
@@ -246,4 +246,24 @@ describe('Auth', () => {
       done();
     });
   });
+
+  it('should return session TTL.', (done) => {
+    spyOn(BBAuth, 'getTTL')
+    .and.returnValue(Promise.resolve(1234));
+
+    BBAuth.getTTL().then((ttl: number) => {
```

Comment:
This test and the one below it don't actually validate anything; you would need to spy on the methods that `BBAuth.getTTL()` calls to ensure that the behavior in that method is what's expected.

Reply:
I have updated the tests to actually test the functions, sorry about that. 

To your general question: We expect consumers of these new methods to be interacting with auth-client directly via their app, rather than through a SKY UX SPA. Right now, this is intended specifically for k12, but will work for other products in a similar situation.

Reply:
I'm not too sure what to make of that code coverage report since I created tests for all the code I added. It looks like it is saying this PR impacted code coverage for files I didn't touch. Should I be doing something about that?